policyengine_us/variables/gov/states/mt/tax/income/deductions/standard/mt_standard_deduction_indiv.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class mt_standard_deduction_indiv(Variable):
+    value_type = float
+    entity = Person
+    label = (
+        "Montana standard deduction when married couples are filing separately"
+    )
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        filing_status = person.tax_unit(
+            "state_filing_status_if_married_filing_separately_on_same_return",
+            period,
+        )
+        p = parameters(period).gov.states.mt.tax.income.deductions.standard
+        agi = person("mt_agi", period)
+        # standard deduction is a percentage of AGI that
+        # is bounded by a min/max by filing status.
+        min_amount = p.min[filing_status]
```

Comment:
suggest renaming parameters
```suggestion
        floor = p.floor[filing_status]
        cap = p.cap[filing_status]
```